File: clang/B4_10987.c
//// https://www.acmicpc.net/problem/10987

#include <stdio.h>
#include <string.h>

int main(void)
{
	char str[101];
	scanf("%s", str);
	
	int i, resp = 0, len = strlen(str);
	for(i = 0; i < len; i ++)
	{
		switch (str[i])
		{
			case 'a':
			case 'e':
			case 'i':
			case 'o':
			case 'u':
				resp++;
		}
	}
	printf("%d", resp);
	return 0;
}

File: go/B4_10987.go
// // https://www.acmicpc.net/problem/10987
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	rw := bufio.NewReadWriter(bufio.NewReader(os.Stdin), bufio.NewWriter(os.Stdout))
	defer rw.Flush()

	var str string
	fmt.Fscan(rw, &str)

	resp := 0
	for i := 0; i < len(str); i++ {
		switch str[i] {
		case 'a', 'e', 'i', 'o', 'u':
			resp++
		}
	}
	fmt.Fprintf(rw, "%d\n", resp)
}
